src/excmds.ts

```diff
@@ -2331,36 +2331,27 @@ export async function tabclose(...indexes: string[]) {
     return done
 }
 
-/** Close all tabs to the right of the current one
+/** Close all tabs to the side specified
  *
  */
 //#background
-export async function tabclosealltoright() {
+export async function tabcloseallto(side: string) {
     const tabs = await browser.tabs.query({
         pinned: false,
         currentWindow: true,
     })
 
     const atab = await activeTab()
+    if (side == "right") {
```

Comment:
Instead of this `if` statement I'd probably do something like this to further reduce duplication:

```js
if !(["left","right"]).includes(side)) throw("side must be left or right")
const comp = side == "right" ? tab => tab.index > atab.index : tab => tab.index < atab.index
const ids = tabs.filter(comp).map(tab => tab.id)
```